src/js/services/pinterest.js

```diff
@@ -9,7 +9,7 @@ module.exports = function(shariff) {
         title += ' - ' + creator;
     }
     var img = shariff.getOption('mediaUrl');
-    if (img && img.length <= 0) {
+    if (!img || img.length <= 7) {
```

Comment:
Yes. I'm also fine with < 11 or 13 if we say it should be a jpg or png or so. It was just a fast fix for a System with wrong output of null.